components/atom/pinInput/demo/ArticleChildren.js

```diff
@@ -91,55 +91,55 @@ const ArticleChildren = ({className}) => {
           <PinInput onChangeHandler={onChangeHandler} defaultValue={cardCode}>
             <Grid cols={19} gutter={[8, 8]}>
               <Cell>
-                <PinInputField isFullWidth />
+                <PinInputField isFullWidth={false} />
```

Comment:
```suggestion
                <PinInputField/>
```

Reply:
I suggest not including the optional `Prop` at all

Reply:
@turolopezsanabria got it! will try it out

Reply:
@turolopezsanabria without including any prop i'll have to make another component for this case, right?

Reply:
@turolopezsanabria i've committed the changes without making new component.